### PR TITLE
Install using npm

### PR DESCRIPTION
**Pull request by okasha93**
Installation guide using NPM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@ stats-lite
 
 A fairly light statistical package. Works with numeric arrays, and will automatically filter out non-numeric values and attempt to convert string numeric values.
 
+Install
+npm install node-stats-lite --save
```
Comment (brycebaril) [inline README.md]: can you add "block quotes" round this line, e.g.

```
npm install node-stats-lite --save
```

**Comment by brycebaril** [comment]
Hi @okasha93! Thank you for your contribution, please see my note above and then I'll get it merged in :)

**Comment by okasha93** [comment]
@brycebaril thanks a lot for this library, edits made to readme file ;) .
sorry for being late

**Comment by brycebaril** [comment]
Hi @okasha93 -- I noticed one more thing; the `npm` installation instructions are `npm install stats-lite --save` ("stats-lite" vs "node-stats-lite")

It turns out `node-stats-lite` on `npm` is a very different package!

**Comment by brycebaril** [comment]
Merged and published as v2.1.1 thanks!

**Pull request merged**